tests/TestCase/TestSuite/TestCaseTest.php

```diff
@@ -361,7 +361,7 @@ public function testGetMockForModel()
 
         $this->assertInstanceOf('TestApp\Model\Table\PostsTable', $Posts);
         $this->assertNull($Posts->save($entity));
-        $this->assertNull($Posts->getTable());
+        $this->assertSame('', $Posts->getTable());
```

Comment:
Should that throw exception instead?

Reply:
This is testing model instance returned by `TestCase::getMockForModel()`. I think it's intended that is behaves a bit differently.

Reply:
Or should we better keep nullable here?